plantcv/plantcv/annotate/classes.py

```diff
@@ -0,0 +1,169 @@
+        self.img = img
+        self.points = {}
+        self.colors = {}
+        self.count = {}  # a dictionary that saves the counts of different classes (labels)
+        self.figsize = figsize
+        self.events = []
+        self.label = None  # current label
+        self.color = None  # current color
+        self.view_all = None  # a flag indicating whether or not view all labels
+        self.fig = None
+        self.ax = None
+        self.p_not_current = None
+
+    def import_coords(self, coords, label="total"):
+        """Import center coordinates of already detected objects
+        Inputs:
+        coords = list of center coordinates of already detected objects.
+        label = class label for imported coordinates, by default label="total".
+
+        Returns:
+        :param coords: list
+        :param label: string
+        :return:
+        """
+        if label not in self.points:
+            self.points[label] = []
+            for (y, x) in coords:
+                self.points[label].append((x, y))
+            self.count[label] = len(self.points[label])
+
+        else:
+            warn(f"{label} already included and counted, nothing is imported!")
+
+    def save_coords(self, coord_file):
+        """Save collected coordinates to a file.
+        Input variables:
+        coord_file = Name of the file to save collected coordinate
+        :param coord_file: str
+        """
+        # Open the file for writing
+        with open(coord_file, "w") as fp:
+            # Save the data in JSON format with indentation
+            json.dump(obj=self.points, fp=fp, indent=4)
+
+    def view(self, label="total", color="c", view_all=False):
+        """
+        View the label for a specific class label
+        Inputs:
+        label = (optional) class label, by default label="total"
+        color = desired color, by default color="c"
+        view_all = indicator of whether view all classes, by default view_all=False
+
+        :param label: string
+        :param color: string
+        :param view_all: boolean
+        :return:
+        """
+        if label not in self.points and color in self.colors.values():
+            warn("The color assigned to the new class label is already used, if proceeding, "
+                 "items from different classes will not be distinguishable in plots!")
+
+        self.label = label
+        self.color = color
+        self.view_all = view_all
+
+        if label not in self.points:
+            self.points[label] = []
+            self.count[label] = 0
+        self.colors[label] = color
+
+        self.fig, self.ax = plt.subplots(1, 1, figsize=self.figsize)
+
+        self.events = []
+        self.fig.canvas.mpl_connect('button_press_event', self.onclick)
+
+        self.ax.imshow(cv2.cvtColor(self.img, cv2.COLOR_BGR2RGB))
+        self.ax.set_title("Please left click on missing pollens\n Right click on those you want to remove")
```

Comment:
Do we want to make this figure title more generalizable than just pollen workflows?